_test/test_experiment/test_experiment_methods.m

```diff
@@ -90,7 +90,7 @@ function test_to_from_old_structure_single_head(obj)
             assertEqual(rec_exp.runid_map.values,{1});
 
             % these properties are only partially recovered from headers
-            assertEqual(rec_exp.samples{1},IX_samp('',[1.1,2.2,3.2],[90,91,92]));
+            assertEqual(rec_exp.samples{1},IX_null_sample('',[1.1,2.2,3.2],[90,91,92]));
             % instrument is not recovered from headers
```

Comment:
`IX_samp` contains lattice. `IX_null_sample` contains no useful information. Have this convention changed and what does this test change mean?

Reply:
It is some misunderstanding here. `IX_samp` and `IX_inst` is not abstract and I remember instantiating `IX_samp` when reading binary sqw files v2 (always) and v3 -- without proper sample defined. It is used in transformations and alignment. I am opened its code and it is not abstract. Not sure about `IX_inst`. It is not used without instrument defined (so full fledged v3 `sqw`), so it may be abstract if no instrument is available. Not sure what is actually in the code. 

your definition defines the idea of `Null_sample` If it is null, you can not use it.